doc(functions): update serverless yml parameters list (#1194)
scaleway/docs-content: serverless/functions/api-cli/fun-uploading-with-serverless-framework.mdx

The different parameters are:
- `service`: your namespace name
- `useDotenv`: load environment variables from `.env` files (default: false), read [Security and secret management](#security-and-secret-management)

> - `configValidationMode`: Configuration validation: 'error' (fatal error), 'warn' (logged to the output) or 'off' (default: warn)

Reviewer: ```suggestion
- `configValidationMode`: configuration validation: 'error' (fatal error), 'warn' (logged to the output) or 'off' (default: warn)
```

- `provider.runtime`: the runtime of your functions ([check the supported runtimes](/serverless/functions/reference-content/functions-lifecycle/))
- `provider.env`: the environment variables attached to your namespace are injected into all your namespace containers
- `provider.secret`: secret environment variables attached to your namespace are injected to all your namespace functions, see [this example project](./examples/secrets)

> - `scwToken`: the Scaleway **API token** you got in prerequisites

Reviewer 2: ```suggestion
- `scwToken`: the Scaleway **API token** you got in the requirements section
```

- `scwProject`: the Scaleway **Project ID** 
- `scwRegion`: the Scaleway region in which the deployment will take place (default: `fr-par`)
- `package.patterns`: usually, this parameter does not need to be configured. It allows you to include or exclude directories from the deployment

> - `functions`: Configuration of your functions. It is a `yml` dictionary, with the key being the functions' name

Reviewer: ```suggestion
- `functions`: configuration of your functions. It is a `yml` dictionary, with the key being the functions' name
```

  - `handler` (Required): file or function which will be executed. 
  - `env` (Optional): environment variables specifics for the current function
  - `secret` (Optional): secret environment variables specific to the current function
  - `minScale` (Optional): how many function instances we keep running (default: 0)
  - `maxScale` (Optional): maximum number of instances this function can scale to (default: 20)
  - `memoryLimit`: RAM allocated to the function instances. 
  - `timeout`: is the maximum duration in seconds that the request will wait to be served before it times out (default: 300 seconds)
  - `runtime`: (Optional) runtime of the function. Can be used when you need to deploy multiple functions with different runtimes in your Serverless Project. If absent, `provider.runtime` will be used.
  - `events` (Optional): List of events that trigger your functions (e.g, trigger a function based on a schedule with `CRONJobs`). 

>   - `custom_domains` (Optional): List of custom domains, refer to the [how to add a custom domain](/serverless/functions/how-to/add-a-custom-domain-name-to-a-function/) documentation page

Reviewer: ```suggestion
  - `custom_domains` (Optional): list of custom domains, refer to the [how to add a custom domain](/serverless/functions/how-to/add-a-custom-domain-name-to-a-function/) documentation page
```
